src/main/java/com/google/sps/servlets/AnswerQuestionServlet.java

```diff
@@ -76,10 +92,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DocumentReference questionDocRef = firestoreDb.collection("games").document(gameId).collection("questions")
         .document(questionId);
 
+    boolean isAnswerCorrect;
 
-    String correctAnswerId = getCorrectAnswer(questionId, questionDocRef);
-
-    boolean isAnswerCorrect = answerId.equals(correctAnswerId);
+    //If there was no answer it's wrong
+    if(answerId == null || answerId.isEmpty()){
```

Comment:
In which cases can answerId be null and in which case can it be an empty string?

Reply:
Oh I think I know why, if the user runs out of time then the answer would be an empty string right?

Reply:
Exactly